statsd-exporter/include/mappings.yml

```diff
@@ -1,19 +1,20 @@
+# For metrics with labels: {},
+# Airflow-instance-specific information like dag_id and task_id
+# are omitted to preserve low cardinality in the metrics.
+# The only label will be the Airflow deployment name,
+# and this may be inserted by Prometheus recording rules
+# in the case of users with many Airflow instances being monitored
+
 mappings:
   # Map dot separated stats to labels
-  - match: airflow.dagrun.dependency-check.*.*
-    name: "airflow_dagrun_dependency_check"
-    labels:
-      dag_id: "$1"
   - match: airflow.operator_successes_(.*)
     match_type: regex
     name: "airflow_operator_successes"
-    labels:
-      operator: "$1"
+    labels: {}
   - match: airflow.operator_failures_(.*)
     match_type: regex
     name: "airflow_operator_failures"
-    labels:
-      operator: "$1"
+    labels: {}
```

Comment:
These ones are used currently:

![image](https://user-images.githubusercontent.com/34150/87434742-c2d15180-c5e2-11ea-80fd-6f16aa29a2ee.png)

Reply:
@ashb I think we have to sacrifice this feature intentionally. For example, one customer is using custom-named operators like CustomerNamePythonOperator_asdf23213 and there are hundreds of them. This is causing the UI to freeze and prometheus to overload on the backend. :/

Reply:
Before we make this change the UI will need to be updated, won't it?

Reply:
I happen to have worked with @samblackk on this yesterday. I will instead show:
"Value: x.xxxxx ops"

Reply:
Alright, next question: is this a config we only want to apply in our cloud, but for enterprise installs it can stay with the current, more granular, labels?

Reply:
That's a good question, and it is up for discussion. My thought is that high cardinality will be seen as a bug by experienced enterprises. In particular, users consuming AC but not enterprise with the helm chart and their own metrics set up I think will desire this behavior.

I think we ought to default to a performant and scalable configuration. In particular dag_id and task_id will very quickly become too large for the average user but operator might not for the average user consuming only normal operators.

Reply:
Dag_id+task_id I can see very quickly getting out of hand.

dag_id alone _might_ be okay, and for _most_ deployments, the operator one will be very low cardinality.

Reply:
what do you think ash?

Reply:
I agree this should be valid most of the time, and it's a fairly nice to have label in those cases. I'm going to first check for a best of both worlds option like truncating cardinality somehow to see if that's possible (PythonOperator, BashOperator, ...., and Other) or yes-listing a set of operators, then the other fall into "other" or something like that.

Reply:
I'm not finding a config option for this purpose, so i'm going to check on the airflow side to see if this could be done

Reply:
it should doable e.g. here https://github.com/astronomer/airflow/blob/dd6aa13a805bbb3e2a180758f5cb572590f37950/airflow/models/taskinstance.py#L1149 and we could also have an optional configuration for allow list in the airflow config. If not specified then any operator name and if specified only a the configured operator names WDYT @ashb, should I proceed down this path of Airflow change?

Reply:
https://github.com/apache/airflow/issues/9823

Reply:
LGTM for now.

Let's see if we can add some of this granularity back later.